python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -416,70 +412,19 @@ def _exists_local(self, path):
         if not super(IODescriptorAppStore, self)._exists_local(path):
             return False
 
-        # Now that we are guaranteed there is a folder on disk, we'll attempt to do some integrity
-        # checking.
-
-        # The metadata folder is a folder that lives inside the bundle.
-        metadata_folder = self._get_metadata_folder(path)
-
-        # If the metadata folder does not exist, this is a bundle that was downloaded with an older
-        # core. We will have to assume that it has been unzipped correctly.
-        if not os.path.isdir(metadata_folder):
-            log.debug(
-                "Pre-core-0.18.80 AppStore download found at '%s'. Assuming it is complete.", metadata_folder
-            )
-            return True
-
-        # Great, we're in the presence of a bundle that was downloaded with integrity check logic.
-
-        # The completed file flag is a file that gets written out after the bundle has been
-        # completely unzipped.
-        completed_file_flag = os.path.join(metadata_folder, self._DOWNLOAD_TRANSACTION_COMPLETE_FILE)
-
-        # If the complete file flag is missing, it means the download operation failed, so we'll
-        # consider it as inexistent.
-        if os.path.exists(completed_file_flag):
+        if os.path.exists(path):
```

Comment:
what is is we check here that the base class isn't checking? Please add a comment to explain